app/src/main/java/com/garethevans/church/opensongtablet/TextSongConvert.java

```diff
@@ -115,7 +115,8 @@ private Resources getLocalizedResources(Context context, Locale desiredLocale) {
 
     private String fixChordLines(String l) {
         // Look for chord lines
-        if (!l.startsWith(".") && !l.startsWith("[") && !l.startsWith(";") && !l.startsWith("-")) {
+        // IV - Do not fix lines with with '_' - this can be added to force a lyric line
+        if (!l.startsWith(".") && !l.startsWith("[") && !l.startsWith(";") && !l.startsWith("-") && (!l.contains("_"))) {
```

Comment:
_ to force a lyric line?  Is that right?

Reply:
During autofix a line can be misidentified as a chord line. Adding a '_' to a line now stops it being identified as a chord line. 
Helpful when editing up a song. Maybe!  This and the [] at the end of line came out of the pain experienced when editing up songs.